website/docs/tutorials/08-cairo-1-support/03-cheatcodes/deploy_contract_cairo.md

```diff
@@ -0,0 +1,35 @@
+# `deploy_contract_cairo0`
+
+```cairo
+fn deploy_contract_cairo0(contract: felt252, calldata: Array::<felt252>) -> 
+Result::<felt252, RevertedTransaction>
+```
+
+Declares and deploys a cairo0 contract given its name defined in the [protostar.toml](../../04-configuration-file.md) configuration
+file.
+
+```cairo title="Example"
+use result::ResultTrait;
+use array::ArrayTrait;
+
+fn my_function() {
+
+    let mut calldata = ArrayTrait::new();
+    calldata.append(10);
+    calldata.append(11);
+    calldata.append(12);
+
+    let address = deploy_contract_cairo0('mycontract', calldata).unwrap();
+
+    // ...
+}
+```
+
+
+Errors raised by the constructor can be handled in the same way as for [deploy](deploy.md)
+
+
+:::info
+`deploy_contract_cairo0` is just a syntactic sugar over executing cheatcodes `declare_cairo0` -> `prepare` -> `deploy` separately, and it's what it does under the hood.
```

Comment:
It's not really a syntactic sugar 
```suggestion
`deploy_contract_cairo0` is just a function, which chains cheatcodes `declare_cairo0` -> `prepare` -> `deploy` under the hood.
```

Reply:
calls instead of chains, as those are not techincally chained